index.js

```diff
@@ -46,6 +55,20 @@ async function setupBazelisk() {
   core.endGroup()
 }
 
+async function isBazelAvailable() {
+  try {
+    await execAsync('bazelisk version')
```

Comment:
It is better to use https://github.com/actions/toolkit/blob/main/packages/exec/ to have cross-platform support (e.g this would fail on Windows because it might need explicit `exe` extension).